### PR TITLE
Clean mod import

### PR DESCRIPTION
**Pull request by mosfet80**
Thank you for creating a pull request to contribute to FreeCAD! Place an "X" in between the brackets below to "check off" to confirm that you have satisfied the requirement, or ask for help in the [FreeCAD forum](https://forum.freecadweb.org/viewforum.php?f=10) if there is something you don't understand.

- [ ]  Your Pull Request meets the requirements outlined in section 5 of [CONTRIBUTING.md](https://github.com/FreeCAD/FreeCAD/blob/master/CONTRIBUTING.md) for a Valid PR

Please remember to update the Wiki with the features added or changed once this PR is merged.  
**Note**: If you don't have wiki access, then please mention your contribution on the [1.0 Changelog Forum Thread](https://forum.freecad.org/viewtopic.php?f=10&t=69438).

---

### CODE REVIEW

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_8858">pipeline status</a> for feature branch [PR_8858](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_8858). Pipeline [803714382](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/803714382) was triggered at [9b0b6ec](https://github.com/FreeCAD/FreeCAD/pull/8858/commits/9b0b6ec780290c2f1eb813be0b7c395c6c0c05ce). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

File: src/Mod/Import/App/AppImportPy.cpp [hunk 1]
```diff
@@ -243,14 +238,10 @@ class Module : public Py::ExtensionModule<Module>
             if (mode >= 0)
                 ocaf.setMode(mode);
             ocaf.loadShapes();
-#elif 1
+
```
Comment (wwmayer) [inline src/Mod/Import/App/AppImportPy.cpp:241]: The code inside the `#elif 1` must also be removed as otherwise it causes a build failure.

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_8858">pipeline status</a> for feature branch [PR_8858](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_8858). Pipeline [803747906](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/803747906) was triggered at [86e991c](https://github.com/FreeCAD/FreeCAD/pull/8858/commits/86e991cb2a3db989f53d161b256791b77f80a4cf). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

**Comment by donovaly** [comment]
Can you please squash the commits to one?

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_8858">pipeline status</a> for feature branch [PR_8858](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_8858). Pipeline [804714910](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/804714910) was triggered at [54946a1](https://github.com/FreeCAD/FreeCAD/pull/8858/commits/54946a1bcab18662d6569a37acf99afb95763fb9). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_8858">pipeline status</a> for feature branch [PR_8858](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_8858). Pipeline [804720917](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/804720917) was triggered at [eb54367](https://github.com/FreeCAD/FreeCAD/pull/8858/commits/eb5436712eb200000709bf1f0db8fd5b43d56898). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

**Comment by donovaly** [comment]
Thanks.

**Pull request merged**